docs/framework/guides/package-generator/typescript-package.md

```diff
@@ -0,0 +1,256 @@
+npm run start -- --language=de
+```
+
+### `test`
+
+Allows executing unit tests for the package, specified in the `tests/` directory. The command accepts the following modifiers:
+
+* `--coverage` &ndash; to create the code coverage report,
+* `--watch` &ndash; to observe the source files (the command does not end after executing tests),
+* `--source-map` &ndash; to generate source maps of the sources,
+* `--verbose` &ndash; to print additional webpack logs.
+
+Examples:
+
+```bash
+# Execute tests.
+npm run test
+
+# Generate code coverage report after each change in the sources.
+npm run test -- --coverage --test
+```
+
+### `lint`
+
+Runs ESLint, which analyzes the code (all `*.ts` files) to quickly find problems.
+
+Examples:
+
+```bash
+# Execute ESLint.
+npm run lint
+```
+
+### `stylelint`
+
+Similar to the `lint` task, stylelint analyzes the CSS code (`*.css` files in the `theme/` directory) in the package.
+
+Examples:
+
+```bash
+# Execute stylelint.
+npm run stylelint
+```
+
+### `dll:build`
+
+Creates a DLL-compatible package build which can be loaded into an editor using {@link installation/advanced/dll-builds DLL builds}.
+
+Examples:
+
+```bash
+# Build the DLL file that is ready to publish.
+npm run dll:build
+
+# Build the DLL file and listen to changes in its sources.
+npm run dll:build -- --watch
+```
+
+### `dll:serve`
+
+Creates a simple HTTP server (without the live-reload mechanism) that allows verifying whether the DLL build of the package is compatible with the CKEditor 5 {@link installation/advanced/dll-builds DLL builds}.
+
+Examples:
+
+```bash
+# Starts the HTTP server and opens the browser.
+npm run dll:serve
+```
+
+<info-box hint>
+    You can run `npm run dll:build -- --watch` and `npm run dll:serve` in two separate command terminals. That way, after you save your changes and reload the page, the content will update.
+</info-box>
+
+### `translations:collect`
+
+Collects translation messages (arguments of the `t()` function) and context files, then validates whether the provided values do not interfere with the values specified in the `@ckeditor/ckeditor5-core` package.
+
+The task may end with an error if one of the following conditions is met:
+
+* Found the `Unused context` error &ndash; entries specified in the `lang/contexts.json` file are not used in source files. They should be removed.
+* Found the `Context is duplicated for the id` error &ndash; some of the entries are duplicated. Consider removing them from the `lang/contexts.json` file, or rewrite them.
+* Found the `Context for the message id is missing` error &ndash; entries specified in the source files are not described in the `lang/contexts.json` file. They should be added.
+
+Examples:
+
+```bash
+npm run translations:collect
+```
+
+### `translations:download`
+
+Download translations from the Transifex server. Depending on users' activity in the project, it creates translations files used for building the editor.
```

Comment:
The dependence between users' activity and the creation of files is unclear. Is it along the lines "if anyone did the translations, they will be downloaded"? This is the same case in both JS and TS files.